utils/namespace.ts

```diff
@@ -7,9 +7,18 @@ const namespaceFromGit = async (): Promise<string | undefined> => {
     return undefined;
   }
   const fetchUrlLine = lns[1];
-  const [_ignoreFetchUrl, _ignoreGitUrl, nsAndGit] = fetchUrlLine.split(":");
-  const [namespace] = nsAndGit.split(".");
-  return namespace.trimEnd();
+  let regex = /.*git@github\.com:\/(.+)/;
+  if (fetchUrlLine.includes("https://")) {
+    // HTTPS
+    // Fetch URL: https://github.com/deco-sites/example
+    regex = /.*https:\/\/github\.com\/(.+)/;
+  }
+
+  const result = fetchUrlLine.match(regex);
+  if (!result || (result.length < 1)) {
+    return undefined;
+  }
+  return "/" + result[1].trimEnd();
```

Comment:
Not sure but I think the namespace should not start with `/` it should be something like `deco-sites/fashion`

Reply:
I kept the current implementation, but it breaks import_map as is.